types/artifacts.d.ts

```diff
@@ -179,6 +181,21 @@ declare global {
         params: {name: string; value: string}[];
       }
 
+      export interface IFrameElement {
+        /** The `id` attribute of the IFrame. */
```

Comment:
I feel like people usually always go lowercase, so my vote is for `iframe` for all these comments (though variable/type names are fine)